feat(changelog): elastic-metal-added-new-beryllium-offers 2026-04-22 (#6454)
scaleway/docs-content: changelog/april2026/2026-04-22-elastic-metal-added-new-beryllium-offers.mdx

---
title: New Beryllium offers
status: added
date: 2026-04-22
category: bare-metal
product: elastic-metal
---

> Introducing the new Epyc 4005 servers in Beryllium range. 

Reviewer: ```suggestion
New Epyc 4005 servers are now available in the Beryllium range with the specifications listed below.
```

> You can find detailed specifications on the configurations below:

Reviewer: ```suggestion
```

> | Offer | Specifications | Price |

Reviewer: ```suggestion
| Offer | CPU | RAM | Storage | Bandwidth | Price |
```

> |---|---|---|

Reviewer: ```suggestion
|---|---|---|---|---|---|
```

> | EM-B130E-NVMe | AMD EPYC 4245P // 6C / 12T - 3.9 GHz // 32 to 64 GB RAM // 2 x 1.02 TB NVMe // 3 to 10 Gbps bandwidth | [See pricing page](https://www.scaleway.com/en/pricing/elastic-metal/) |

Reviewer: ```suggestion
| EM-B130E-NVMe | AMD EPYC 4245P (6C / 12T - 3.9 GHz) | 32 to 64 GB RAM | 2 x 1.02 TB NVMe | 3 to 10 Gbps bandwidth | [See pricing page](https://www.scaleway.com/en/pricing/elastic-metal/) |
```

> | EM-B230E-NVMe | AMD EPYC 4345P // 8C / 16T - 3.8 GHz // 64 to 128 GB RAM // 2 x 1.02 TB NVMe // 3 to 10 Gbps bandwidth | [See pricing page](https://www.scaleway.com/en/pricing/elastic-metal/) |

Reviewer: ```suggestion
| EM-B230E-NVMe | AMD EPYC 4345P (8C / 16T - 3.8 GHz) | 64 to 128 GB RAM | 2 x 1.02 TB NVMe | 3 to 10 Gbps bandwidth | [See pricing page](https://www.scaleway.com/en/pricing/elastic-metal/) |
```

> | EM-B330E-NVMe | AMD EPYC 4465P // 12C / 24T - 3.4 GHz // 64 to 128 GB RAM // 2 x 2.05 TB NVMe // 3 to 10 Gbps bandwidth | [See pricing page](https://www.scaleway.com/en/pricing/elastic-metal/) |

Reviewer: ```suggestion
| EM-B330E-NVMe | AMD EPYC 4465P (12C / 24T - 3.4 GHz) | 64 to 128 GB RAM | 2 x 2.05 TB NVMe | 3 to 10 Gbps bandwidth | [See pricing page](https://www.scaleway.com/en/pricing/elastic-metal/) |
```

> | EM-B430E-NVMe | AMD EPYC 4545P // 16C / 32T - 3.0 GHz // 64 to 128 GB RAM // 2 x 3.84 TB NVMe // 3 to 10 Gbps bandwidth | [See pricing page](https://www.scaleway.com/en/pricing/elastic-metal/) |

Reviewer: ```suggestion
| EM-B430E-NVMe | AMD EPYC 4545P (16C / 32T - 3.0 GHz) | 64 to 128 GB RAM | 2 x 3.84 TB NVMe | 3 to 10 Gbps bandwidth | [See pricing page](https://www.scaleway.com/en/pricing/elastic-metal/) |
```

> | Available in PAR-1, PAR-2 and AMS-1 |

Reviewer: ```suggestion

The servers are available in the following regions: PAR-1, PAR-2, and AMS-1.
```
